stubs/netaddr/netaddr/ip/sets.pyi

```diff
@@ -0,0 +1,47 @@
+from _typeshed import Self
+from collections.abc import Generator, Iterable, Iterator
+from typing import NoReturn
+from typing_extensions import TypeAlias
+
+from netaddr.ip import IPAddress, IPNetwork, IPRange, _IPNetworkAddr
+
+_IPIterable: TypeAlias = IPNetwork | IPRange | IPSet | Iterable[_IPNetworkAddr | IPRange | int]
+
+class IPSet:
+    _cidrs: dict[IPNetwork, bool]
```

Comment:
We _generally_ omit private attributes in typeshed, unless there's a specific reason for including them. Is there a specific reason to make an exception here?

```suggestion
```

Reply:
I type some core private attributes so that I get maximum help from vscode (pylance) to validate the types I set. I must've forgotten to remove this one. I'll remove it.